File: src/org/labrad/qubits/resources/BiasBoard.java
package org.labrad.qubits.resources;

import org.labrad.qubits.enums.DcRackFiberId;
import org.labrad.qubits.enums.DacFiberId;

public interface BiasBoard extends Resource {
	public void setDacBoard(DcRackFiberId channel, DacBoard board, DacFiberId fiber);
}

File: QubitServer/src/main/scala/org/labrad/qubits/resources/BiasBoard.scala
package org.labrad.qubits.resources

import org.labrad.qubits.enums.DacFiberId
import org.labrad.qubits.enums.DcRackFiberId

trait BiasBoard extends Resource {
  def setDacBoard(channel: DcRackFiberId, board: DacBoard, fiber: DacFiberId)
}
